packages/eslint-plugin-mobx/src/exhaustive-make-observable.js

```diff
@@ -66,46 +73,61 @@ function create(context) {
       }
       */
 
-      if (!hasAnyDecorator && nonAnnotatedMembers.length) {
-        // Set avoids reporting twice for setter+getter pair or actual duplicates
-        const keys = [...new Set(nonAnnotatedMembers.map(fieldToKey))];
-        const keyList = keys.map(key => `\`${key}\``).join(', ');
+            if (!hasAnyDecorator && nonAnnotatedMembers.length) {
+                // Set avoids reporting twice for setter+getter pair or actual duplicates
+                const keys = [...new Set(nonAnnotatedMembers.map(fieldToKey))]
+                const keyList = keys.map(key => `\`${key}\``).join(", ")
+                const annotation =
+                    typeof autofixAnnotation === "boolean" ? autofixAnnotation : "true"
```

Comment:
Can we move the boolean check to the line 9, where we extract value from options? I would throw if `context.options[0]?.autofixAnnotation` is anything but boolean/undefined.

Reply:
From what I understand of the eslint rule spec, the inputs are validated using the JSON schema added in the `meta.schema` section: https://eslint.org/docs/latest/extend/custom-rules#options-schemas

It should be safe to assume here already that `autofixAnnotation` is always boolean/undefined so the explicit check isn't necessary.